dagger1support/src/main/java/mortar/dagger1support/Blueprint.java

```diff
@@ -5,16 +5,17 @@
 import mortar.MortarScope;
 
 /**
- * Defines a scope to be built via {@link Dagger1#requireChild(MortarScope, Blueprint)} or
- * {@link Dagger1#requireActivityScope(MortarScope, Blueprint)}.
+ * Defines a scope to be built via {@link ObjectGraphService#requireChild(MortarScope, Blueprint)} or
+ * {@link ObjectGraphService#requireActivityScope(MortarScope, Blueprint)}.
  */
+@Deprecated
```

Comment:
Add a `@deprecated` note in the javadoc, explaining when and why and/or what to do instead.